Lib/test/_test_multiprocessing.py

```diff
@@ -5342,6 +5342,16 @@ def test_ignore_listener(self):
         finally:
             conn.close()
 
+# Utility functions used as target for spawn Process
```

Comment:
Note that you can make those classmethods just above the associated test method.